File: MORM.Java/src/MORM/Java/Models/Venda/Caixa.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "CAIXA")
public class Caixa extends CollectionItem
{
    @CampoAttribute(Campo = "ID_CAIXA", Tipo = CampoTipo.Key)
    public Integer Id_Caixa;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "ID_EMPRESA", Tipo = CampoTipo.Req)
    public Integer Id_Empresa;
    @CampoAttribute(Campo = "ID_TERMINAL", Tipo = CampoTipo.Req)
    public Integer Id_Terminal;
    @CampoAttribute(Campo = "DT_ABERTURA", Tipo = CampoTipo.Req)
    public Date Dt_Abertura;
    @CampoAttribute(Campo = "VL_ABERTURA", Tipo = CampoTipo.Req)
    public Float Vl_Abertura;
    @CampoAttribute(Campo = "IN_FECHADO", Tipo = CampoTipo.Req)
    public String In_Fechado;
    @CampoAttribute(Campo = "DT_FECHADO", Tipo = CampoTipo.Nul)
    public Date Dt_Fechado;
}

File: MORM.CSharp/MORM.CSharp/Models/Venda/Caixa.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("CAIXA")]
    public class Caixa : CollectionItem
    {
        [Campo("ID_CAIXA", CampoTipo.Key)]
        public int Id_Caixa { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("ID_EMPRESA", CampoTipo.Req)]
        public int Id_Empresa { get; set; }
        [Campo("ID_TERMINAL", CampoTipo.Req)]
        public int Id_Terminal { get; set; }
        [Campo("DT_ABERTURA", CampoTipo.Req)]
        public DateTime Dt_Abertura { get; set; }
        [Campo("VL_ABERTURA", CampoTipo.Req)]
        public double Vl_Abertura { get; set; }
        [Campo("IN_FECHADO", CampoTipo.Req)]
        public string In_Fechado { get; set; }
        [Campo("DT_FECHADO", CampoTipo.Nul)]
        public DateTime Dt_Fechado { get; set; }
    }
}